pkg/kubelet/volumemanager/reconciler/reconciler.go

```diff
@@ -351,84 +362,76 @@ func (rc *reconciler) reconciliationFunc() {
 	}
 }
 
-func (rc *reconciler) reconstructFunc() {
-	glog.V(1).Infof("Start the routine to reconstruct volumes from disk directories %q", rc.kubeletPodsDir)
-	rc.reconstructFromDisk(rc.kubeletPodsDir)
+// reconstruc process tries to observes the real world by scaning all pods' volumes directories from the disk.
+// If the actual and desired state of worlds are not consistent with the observed world, that means tsome
+// mounted volumes are left out probably during kubelet restart. This process will reconstruct
+// the volumes and udpate the actual and desired states. In the following reconciler loop, those volumes will
+// be cleaned up.
+func (rc *reconciler) reconstruct() {
+	glog.V(1).Infof("Start the routine to reconstruct states by reading volumes dirs from disk directories %q", rc.kubeletPodsDir)
```

Comment:
This doesn't need to be `v(1)`, it'll be printed on every iteration of the loop and is mostly useless and will eat up disk space.